src/util/nodelist-to-array.js

```diff
@@ -0,0 +1 @@
+export default (nodeList) => Array.prototype.slice(nodeList);
```

Comment:
This works for more than node lists, so I wonder if it's worth reflecting that in the name?